jsk_perception/node_scripts/ssd_train_dataset.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
```

Comment:
I think this file is not node.
It should be placed in `scripts/` or somewhere

Reply:
+1

Reply:
@knorth55 @wkentaro True! Moved to `scripts`.

Reply:
Oh no. It is still in `node_scripts` directory. Maybe I forgot to push this commit? If you mind, I'll create another pull request later, sorry.

Reply:
@furushchev I moved it to `script` in #2305